src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -152,42 +151,24 @@ private void maybeInitJmx()
         // then the JVM agent will have already started up a default JMX connector
         // server. This behaviour is deprecated, but some clients may be relying
         // on it, so log a warning and skip setting up the server with the settings
-        // as configured in cassandra-env.(sh|ps1)
+        // as configured in cassandra.yaml
```

Comment:
I think the comment should say in cassandra-env.(sh|ps1) or cassandra.yaml, because my understanding is cassandra-env.sh is still supported?